source/common/filesystem/win32/directory_iterator_impl.cc

```diff
@@ -50,7 +50,8 @@ DirectoryEntry DirectoryIteratorImpl::makeEntry(const WIN32_FIND_DATA& find_data
     return {std::string(find_data.cFileName), FileType::Other, absl::nullopt};
   } else if (find_data.dwFileAttributes & FILE_ATTRIBUTE_DIRECTORY) {
     return {std::string(find_data.cFileName), FileType::Directory, absl::nullopt};
-  } else if (find_data.dwReserved0 & IO_REPARSE_TAG_SYMLINK) {
+  } else if ((find_data.dwFileAttributes & FILE_ATTRIBUTE_REPARSE_POINT) &&
```

Comment:
can you add the link from the description into the source code here?

Reply:
Done.